tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -34,6 +34,7 @@ sub run {
     }
     $variables{OS_OWNER} = get_var('QESAP_CLUSTER_OS_OWNER', 'amazon') if check_var('PUBLIC_CLOUD_PROVIDER', 'EC2');
 
+    $variables{USE_SAPCONF} = get_required_var('USE_SAPCONF');
```

Comment:
Please use setting name following naming convention `QESAPDEPLOY`

```
get_required_var('QESAPDEPLOY_USE_SAPCONF');
```

I try to follow this convention :
 - `QESAPDEPLOY_`  for settings used in test modules that belong to qesap regression
 - `QESAP_` for settings used in `lib/qesapdeployment.pm`

Reply:
Good suggestion. I will follow it next time when introducing new vars.
In this PR, the `USE_SAPCONF` is an old variable so I just use it.